### PR TITLE
Make the messages on spinners more targetted about the kind of operation being performed

### PR DESCRIPTION
**Pull request by nyjalusc**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @nyjalusc has signed the CLA. Thanks for the pull request!

File: box-share-sdk/src/main/java/com/box/androidsdk/share/fragments/SharedLinkAccessFragment.java [hunk 1]
```diff
@@ -355,7 +355,7 @@ public void onDateSet(DatePicker view, int year, int month, int day) {
         // Do something with the date chosen by the user
         GregorianCalendar calendar = new GregorianCalendar(year, month, day);
         try {
-            showSpinner();
+            showSpinner(R.string.box_sharesdk_fetching_collaborators, R.string.boxsdk_Please_wait);
```
Comment (rgcorrea) [inline box-share-sdk/src/main/java/com/box/androidsdk/share/fragments/SharedLinkAccessFragment.java]: shouldn't this also have a message related to shared link update ?

File: box-share-sdk/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -13,7 +13,11 @@
     <string name="box_sharesdk_accessible_collaborator" description="description for collaborator access">Accessible to people the folder is shared with</string>
     <string name="box_sharesdk_password_protected" description="description for if a shared link is password proceted">Password protected</string>
     <string name="box_sharesdk_password_unprotected" description="description for if a shared link is not password proceted">No password</string>
-    <string name="boxsdk_fetching_collaborators" description="message to show on spinner when we are fetching collaborators">Fetching collaborators</string>
+    <string name="box_sharesdk_fetching_collaborators" description="message to show on spinner when we are fetching collaborators">Fetching collaborators</string>
+    <string name="box_sharesdk_adding_collaborators" description="message to show on spinner when we are adding collaborators">Adding collaborators</string>
+    <string name="box_sharesdk_enabling_share_link" description="message to show on spinner when we are enabling share link">Enabling share link</string>
```
Comment (rgcorrea) [inline box-share-sdk/src/main/res/values/strings.xml]: Shouldn't the text be "XXX shared link" ?

Comment (nyjalusc) [inline box-share-sdk/src/main/res/values/strings.xml]: I am trying to be consistent with the wording we use in our app. We use the term "Share Link" everywhere so I thought in the message "Enabling share link" would be more accurate.

**Comment by rgcorrea** [comment]
+1

**Pull request merged**